scipy/odr/tests/test_odr.py

```diff
@@ -563,3 +564,50 @@ def func(b, x):
             odr_obj.set_job(fit_type=0, del_init=1)
             # Just make sure that it runs without raising an exception.
             odr_obj.run()
+
+    # Test pickling
+
+    def test_pickling_data(self):
+        x = np.linspace(0.0, 5.0)
+        y = 1.0 * x + 2.0
+        data = Data(x, y)
+
+        obj_pickle = pickle.dumps(data)
+        del data
+        pickle.loads(obj_pickle)
+        
+
```

Comment:
Style: we only have one blank line between methods inside of a class definition.